### PR TITLE
[Console] Document the option about progress bar starting points

### LINKED ISSUE #17050: [Console] Add support for resuming a ProgressBar
| Q            | A
| ------------ | ---
| Feature PR   | symfony/symfony#46242
| PR author(s) | @yivi
| Merged in    | 6.2

### PR DESCRIPTION
**Pull request by javiereguiluz**
Fixes #17050.

### CODE REVIEW

File: components/console/helpers/progressbar.rst [hunk 1]
```diff
@@ -46,6 +46,22 @@ Instead of advancing the bar by a number of steps (with the
 you can also set the current progress by calling the
 :method:`Symfony\\Component\\Console\\Helper\\ProgressBar::setProgress` method.
 
+If you rea resuming long-standing tasks, it's useful to start drawing the progress
```
Comment (alexisLefebvre) [inline components/console/helpers/progressbar.rst]:
```suggestion
If you are resuming long-standing tasks, it's useful to start drawing the progress
```

File: components/console/helpers/progressbar.rst [hunk 2]
```diff
@@ -46,6 +46,22 @@ Instead of advancing the bar by a number of steps (with the
...
+bar at a certain point. Use the second optional argument of ``start()`` to set
+that starting point::
+
+    use Symfony\Component\Console\Helper\ProgressBar;
+
+    // creates a new progress bar (10,000 units)
```
Comment (alexisLefebvre) [inline components/console/helpers/progressbar.rst]: An example with `100` is enough in my humble opinion, big values as `10000` and `2500` are not necessary in this example.

File: components/console/helpers/progressbar.rst (hunk 1 above)
Comment (javiereguiluz) [reply components/console/helpers/progressbar.rst]: Fixed. Thanks.

File: components/console/helpers/progressbar.rst (hunk 2 above)
Comment (javiereguiluz) [reply components/console/helpers/progressbar.rst]: Yes, I agree on this. I've just updated the example to use `100` instead of `10000`. Thanks!

**Pull request closed without merging**